kale/crypt.py

```diff
@@ -34,10 +34,16 @@
 # Set up our Crypto hashing object
 BLOCK_SIZE = 16
 
+
+def _set_cipher():
+    global cipher
+    cipher = AES.new(builtins.bytes(settings.UTIL_CRYPT_CIPHER, 'ascii'), AES.MODE_ECB)
```

Comment:
Alternatively, we could put a `'b'` in front of this setting, but then this change is not backwards compatible with whatever settings file you might be using.